src/app/activitywidget.cpp

```diff
@@ -223,10 +223,30 @@ void ActivityWidget::updateControls()
         mUi->btStartStop->setText(tr("Start"));
         mUi->btStartStop->setIcon(QIcon(":/icons/play"));
         mUi->lbDurationTime->clear();
+
+        window()->setWindowIcon(QIcon(":/icons/kemai"));
+
+        // TODO: probably not the best way to do this, needs improvement
+        MainWindow* mW = static_cast<MainWindow*>(getMainWindow());
```

Comment:
I like the idea of change icon when activiy starts howewer a Qt signal will be the right things. Then MainWindow can listen to this signals and change its icon.